src/sidebar/test/cross-origin-rpc-test.js

```diff
@@ -46,28 +46,60 @@ describe('crossOriginRPC', function() {
       assert.isTrue(fakeWindow.addEventListener.calledWith('message'));
     });
 
-    it('sends a response with the result from the called method', function() {
+    it('sends a response with the "ok" result', function() {
       crossOriginRPC.server.start(fakeStore, settings, fakeWindow);
 
       postMessage({
-        data: { method: 'searchUris', id: 42 },
+        data: { method: 'changeFocusModeUser', id: 42 },
```

Comment:
This message payload doesn't include params; is this really a valid request? What would be the resulting state of the app if such a request were made?

Reply:
It's valid to call w/o args, there is a condition in the rpc server that checks if there are args, and then destructures them into the method, but, it should be noted that would produce an invalid user. The action still works, the reducer still works and the user ends up being set to  `{}` which then the selectors appropriately do not show focused mode because the `username` is missing. I think thats okay?

Reply:
I went ahead and added one more test to demonstrate very clearly that missing args are okay. I think that test makes sense since it calls that out specifically even those it was technically tested in this function too.